mandelbulber2/src/compute_fractal.cpp

```diff
@@ -110,10 +110,12 @@ void Compute(const cNineFractals &fractals, const sFractalIn &in, sFractalOut *o
 	int i;
 	int sequence = 0;
 
-	CVector3 lastGoodZ;
-	CVector3 lastZ;
+	StructuredCVector3 lastGoodZ;
+	StructuredCVector3 lastZ;
 
-	for (i = 0; i < in.maxN; i++)
+	// chop this loop into sets of 8
+	//   to use StructuredCVector3 and double*
+	for (i = 0; i < in.maxN; i = i + 8)
 	{
 		lastGoodZ = lastZ;
 		lastZ = z;
```

Comment:
Can these loop iterations be subdivided into sets of 8?
The use of `lastGoodZ = lastZ; lastZ = z; `makes me nervous about Loop dependence analysis.
Can we safely execute 8 initial iterations in paralel and step through this loop with increments of 8?
`for (i = 0; i < in.maxN; i = i + 8)`
What are you thoughts?